redirector/go/main.go

```diff
@@ -240,6 +240,22 @@ func (r *root) Lookup(
 }
 
 func main() {
+	if len(os.Args) != 2 {
+		fmt.Fprintf(os.Stderr, "Usage: %s <mountpoint>\n", os.Args[0])
+		os.Exit(1)
+	}
+
+	// Restrict the mountpoint to paths starting with "/keybase".
+	// Since this is a suid binary, it is dangerous to allow arbitrary
+	// mountpoints.  TODO: Read a redirector mountpoint from a
+	// root-owned config file.
+	r := newRoot()
+	if os.Args[1] != fmt.Sprintf("/%s", r.runmodeStr) {
```

Comment:
Maybe this should be case insensitive.

Reply:
@oconnor663 wanted this to be as restrictive as possible.  We can always change it later if we need   to.

Reply:
Understood, thanks.